pkgs/development/interpreters/python/tests/test_environments/test_python.py

```diff
@@ -38,7 +38,7 @@ def test_site_prefix(self):
 
     @unittest.skipIf(IS_PYPY or sys.version_info.major==2, "Python 2 does not have base_prefix")
     def test_base_prefix(self):
-        if IS_VENV or IS_NIXENV or IS_VIRTUALENV:
+        if IS_VENV or IS_VIRTUALENV:
```

Comment:
This is the motivation for this whole PR: a nix environment shouldn't appear to be a venv.